README.md

```diff
@@ -15,15 +15,22 @@ Help would be appreciated! Please join us in [slack #flaredown](https://rubyforg
 * Redis 6.2.3
 * Ruby 2.7.4 (see [RVM](https://rvm.io/) also)
 * Node 12.22.6
+## Installation
 
-You can spin up instances of the required data-stores in Docker containers using `docker compose up` in the project root.
```

Comment:
Did you mean to remove this line?

Reply:
This command does the setup/run, so I thought it would make sense to include it on line 59 of this PR below, where we instruct the developer to spin up the databases.